django_mongodb/fields/json_field.py

```diff
@@ -0,0 +1,160 @@
+from django.db import NotSupportedError
+from django.db.models import JSONField
+from django.db.models.fields.json import (
+    ContainedBy,
+    DataContains,
+    HasAnyKeys,
+    HasKey,
+    HasKeyLookup,
+    HasKeys,
+    JSONExact,
+    KeyTransform,
+    KeyTransformIn,
+    KeyTransformIsNull,
+    KeyTransformNumericLookupMixin,
+)
+
+from django_mongodb.lookups import builtin_lookup
+from django_mongodb.query_utils import process_lhs, process_rhs
+
+
+def _key_transform_root(self, compiler, connection):
```

Comment:
I think it would be better to put this right above the function where it's used. (If it even makes sense to have this as a separate function that's only called in one place?)